ArchiSteamFarm/IPC/Controllers/Api/GitHubController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using ArchiSteamFarm.IPC.Responses;
+using ArchiSteamFarm.Localization;
+using Microsoft.AspNetCore.Mvc;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ArchiSteamFarm.IPC.Controllers.Api {
+	[ApiController]
+	[Route("Api/WWW/GitHub")]
+	public sealed class GitHubController : ControllerBase {
+		[HttpGet("Releases/{version:required}")]
+		public async Task<ActionResult<GenericResponse<GitHubReleaseResponse>>> GetRelease(string version) {
+			if (string.IsNullOrEmpty(version)) {
+				return BadRequest(new GenericResponse<GitHubReleaseResponse>(false, string.Format(Strings.ErrorIsEmpty, nameof(version))));
+			}
+
+			GitHub.ReleaseResponse releaseResponse = await GitHub.GetRelease(version).ConfigureAwait(false);
+			if (releaseResponse == null) {
+				return BadRequest(new GenericResponse<GitHubReleaseResponse>(false, string.Format(Strings.ErrorRequestFailedTooManyTimes, WebBrowser.MaxTries)));
+			}
+
+			return Ok(new GenericResponse<GitHubReleaseResponse>(new GitHubReleaseResponse(releaseResponse)));
+		}
+
+		[HttpGet("Releases")]
+		public async Task<ActionResult<IEnumerable<GitHubReleaseResponse>>> GetReleases([FromQuery] byte? count) {
+			if (count == null) {
+				count = 10;
+			}
+
+			List<GitHub.ReleaseResponse> response = await GitHub.GetReleases(count.Value).ConfigureAwait(false);
```

Comment:
Add check against `count.Value` being `0` which would cause ASF to log null error once again because of **wrong user input**.